third_party/move/move-model/src/builder/module_builder.rs

```diff
@@ -587,9 +593,9 @@ impl<'env, 'translator> ModuleBuilder<'env, 'translator> {
         let module_name = ModuleName::new(given_addr, module_sym);
         let module_id = self
             .parent
-            .env
-            .find_module(&ModuleName::new(resolved_addr, module_sym))
-            .map(|e| e.get_id());
+            .module_table
```

Comment:
`find_module` is a linear search, so using the newly added `module_table` for a lookup instead.